tests/unit/records/marshmallow/experiments/test_base.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from helpers.providers.faker import faker
+from inspire_schemas.api import load_schema, validate
+
+from inspirehep.records.api import ExperimentsRecord
+from inspirehep.records.marshmallow.experiments import ExperimentsMetadataRawSchemaV1
+
+
+def test_experiment_serializer_should_serialize_whole_basic_record():
+    schema = ExperimentsMetadataRawSchemaV1()
+    expected_result = {
+        "$schema": "http://localhost:5000/schemas/records/experiments.json",
+        "_collections": ["Experiments"],
+        "project_type": ["experiment"],
+    }
+
+    experiment = ExperimentsRecord(faker.record("exp"))
```

Comment:
I would put data here explicitly rather than using `faker`s defaults, when you need to assert directly the data. Because now if we change default experiment record in `faker`, everything will fail unnecessarily.

Reply:
If I am not mistaken, we are updating data to fake records after generating fake data, so if we add in the future 'fake' experiment data it will still be overwritten by experiment put in `data` parameter so it should not break anything.

Reply:
But right, here we are not providing any data in this test. I will add empty experiment key in data then.